src/v2/controllers/vreplicaset_controller/proof/helper_invariants/proof.rs

```diff
@@ -489,6 +490,8 @@ pub proof fn lemma_eventually_always_every_create_matching_pod_request_implies_a
         spec.entails(always(lift_state(Cluster::cr_objects_in_reconcile_satisfy_state_validation::<VReplicaSetView>(controller_id)))),
         spec.entails(always(lift_state(Cluster::the_object_in_reconcile_has_spec_and_uid_as(controller_id, vrs)))),
         spec.entails(always(lift_state(Cluster::each_object_in_reconcile_has_consistent_key_and_valid_metadata(controller_id)))),
+        vrs.metadata.deletion_timestamp.is_None(),
```

Comment:
Is line 493 necessary? `vrs` isn't going to reflect this dynamic state correctly anyway.